apps/gk_species.c

```diff
@@ -1156,6 +1189,10 @@ gk_species_new_dynamic(struct gkyl_gk *gk_app_inp, struct gkyl_gyrokinetic_app *
   else
     gks->apply_pos_shift_func = gk_species_apply_pos_shift_disabled;
   gks->write_func = gk_species_write_dynamic;
+  if (gks->info.cfl_dt_diagnostic)
+    gks->write_cfl_func = gk_species_write_cfl_dynamic;
+  else 
+    gks->write_cfl_func = gk_species_write_cfl_static;
```

Comment:
instead of _dynamic and _static, can you use _enabled and _disabled ? I want to start moving to this language to reserve dynamic and static specifically for the distinction between dynamic and static species (or fields)